packages/builder/uni-builder/package.json

```diff
@@ -76,6 +76,7 @@
   },
   "devDependencies": {
     "@modern-js/builder-plugin-node-polyfill": "workspace:*",
+    "@modern-js/server-core": "workspace:*",
```

Comment:
Why was this dependency included?

Reply:
uni-builder need ServerBaseOptions type from modern-js/server-core